interface/patient_file/summary/demographics_full.php

```diff
@@ -1158,7 +1158,10 @@ class='text'>(<?php echo xlt('Browse'); ?>)</a>
             var fval = $('#form_' + flds[i]).val();
             duplicateFieldsArray['#form_' + flds[i]] = fval;
         }
-
+        $(".sel2").select2({
+            <?php require($GLOBALS['srcdir'] . '/js/xl/select2.js.php'); ?>
+        });
```

Comment:
Could try:
```
$(".sel2").select2({
    theme: "bootstrap4",
    dropdownAutoWidth: true,
    width: 'resolve',
<?php require($GLOBALS['srcdir'] . '/js/xl/select2.js.php'); ?>
});
```

Reply:
works like a champ :boxing_glove: 
pushed it to #5352 and now passing cms147v11
![sheep](https://user-images.githubusercontent.com/5618372/168939119-0f604ec0-04e8-4d37-a5d0-ce33a36d4a9c.gif)

Reply:
![munchkin-wizard-of-oz](https://user-images.githubusercontent.com/278968/168948917-293db278-9164-43ba-8201-d1912feb9642.gif)